executable/SummarizeApp.re

```diff
@@ -4,16 +4,32 @@ open AppSchema;
 type role =
   | User
   | Admin;
+
+type summary = {
+  unread: int,
+  total: int,
+  projects: int,
+  organizations: int,
+};
+
 type user = {
   id: int,
   name: string,
   role,
+  summary,
 };
 
-let users = [
-  {id: 1, name: "Alice", role: Admin},
-  {id: 2, name: "Bob", role: User},
-];
+let mockedUser: user = {
+  id: 1,
+  name: "Alice",
+  role: Admin,
+  summary: {
```

Comment:
this actually won't come from the user model, it'll be aggregated just in time (commenting below)